frontend/javascripts/oxalis/view/plane_view.ts

```diff
@@ -248,6 +249,11 @@ class PlaneView {
     this.resize();
     this.animate();
     window.addEventListener("resize", this.resizeThrottled);
+    listenToStoreProperty(
+      (storeState) => storeState.uiInformation.navbarHeight,
+      () => this.resizeThrottled(),
+      true,
+    );
```

Comment:
```suggestion
    this.unsubscribeFunctions.push(listenToStoreProperty(
      (storeState) => storeState.uiInformation.navbarHeight,
      () => this.resizeThrottled(),
      true,
    ));
```

that way, the listener gets removed again on `stop()` (e.g., when switching to arbitrary mode).